src/Input.jsx

```diff
@@ -10,7 +10,7 @@ export default function Input({ value, onChange, onClick }) {
         id="input-task-title"
         type="text"
         placeholder="할 일을 입력해 주세요"
-        value={value}
+        defaultValue={value}
```

Comment:
value 를 변경하신 이유가있으신가요?

구현부를 이렇게 변경하면 테스트가 깨지는 부분이 생길거에요.

테스트 코드가 없다면 인지하지 못할 수도있죠. 이런 부분에서 테스트 코드의 장점을 알 수 있어요 ;)

Reply:
```
Warning: Failed prop type:
You provided a `value` prop to a form field without an `onChange` handler.
This will render a read-only field. If the field should be mutable use `defaultValue`.
Otherwise, set either `onChange` or `readOnly`.
```
이런 에러가 나서 아래의 글을 참고해서 변경하였습니다.
https://velog.io/@hwang-eunji/react-Input-%ED%83%9C%EA%B7%B8-%EC%86%8D%EC%84%B1-value-defaultValue

그런데 또 지금 value로 바꿔보니 에러가 안나네요... 왜 났던건지 모르겠습니다 ㅜㅜ 수정할게요...!

Reply:
에러 메시지가 자세하고도 정확하게 에러를 말해주고 있어요. 

다만 제 생각으로는 `defaultValue` 는 의도를 생각했을 때 좋은 해결책은 아닌 것 같아요.

Reply:
`onChange` 핸들러 없이 `input`에 `value`를 수정해서 위와같은 에러가 난다고 이해했는데요.

아래의 코드에서 value를 바꿔주었기때문에 에러가 나는건가요?..
```js
fireEvent.change(getByLabelText('할 일'), { target: { value: '홈트하기' } });
```

Reply:
테스트 코드는 잘 작성하셨어요.

코드를 복사해서 똑같이 테스트를 돌려봐도 저는 에러가 발생하지 않는데 아직도 계속 에러가 발생하나요?

`Input` 컴포넌트를 수정하신게 아니라면 테스트상으로는 문제가 없어보여요.

만약에 에러가 발생한다고 해도 `defaultValue` 를 사용하지 마시고 그대로 `value` 를 사용해서 해결해보세요!

테스트를 위해 구현부를 고치는 일은 없어야해요